packages/dicomImageLoader/src/imageLoader/wadouri/metaData/metaDataProvider.ts

```diff
@@ -245,6 +249,10 @@ function metaDataProvider(type, imageId) {
       actualFrameDuration: dicomParser.intString(dataSet.string('x00181242')),
     };
   }
+
+  if (type === 'instance') {
+    return getInstanceModule(imageId, metaDataProvider, instanceModuleNames);
```

Comment:
Not sure if it would make sense to use the `naturalizeDataset` or `namifyDataset` methods of `dcmjs` [DicomMetaDictionary](https://github.com/dcmjs-org/dcmjs/blob/f844e025f86055a918c13269f6d762a711f3a4bf/src/DicomMetaDictionary.js#L111) to get the full metadata of the instance that already match the dcmjs naming.

Reply:
the problem is that we don't have the instance stored anywhere to naturalize or namify it.